Sources/Architecture/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -129,7 +129,11 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
 #if defined(ARCH_ARM)
   case 2: // udf #1
     opcode += '\xde';
+#if defined(OS_POSIX)
     opcode += '\x01';
+#elif defined(OS_WIN32)
+    opcode += '\xfe';
```

Comment:
Never realized that before. Makes sense :p

On Wednesday, June 8, 2016, walter erquinigo <notifications@github.com>
wrote:

> In Sources/Architecture/ARM/SoftwareBreakpointManager.cpp
> <https://github.com/facebook/ds2/pull/363#discussion_r66322126>:
>
> >      opcode += '\x01';
> > +#elif defined(OS_WIN32)
> > +    opcode += '\xfe';
>
> i've just realized that 01 = ~fe
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/363/files/7cae76220b2c9ed7e762a314ac98f2cb98bdef53..d1f5acb1d4ec737a2de2218a5ae498a34dead41b#r66322126>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbAUiQHron-48Yvht8f2XRYLtKloEks5qJxbDgaJpZM4IwaBw>
> .
>


-- 
Stephane Sezer